packages/devtools-reps/src/reps/tests/array.js

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { shallow } = require("enzyme");
+const React = require("react");
+
+const {
+  REPS,
+  getRep,
+} = require("../rep");
+
+const { MODE } = require("../constants");
+const { ArrayRep, Rep } = REPS;
+
+const maxLength = {
+  short: 3,
+  long: 10
+};
```

Comment:
nit: Could we use the same approach as for GripArray? I.e. exporting map of max lengths for individual modes and reusing it?